cluster.go

```diff
@@ -217,7 +217,7 @@ func (c *Cluster) handleSerfEvent(e serf.Event) {
 
 	case serf.EventMemberFailed:
 		c.logger.Debug("Handling unresponsive member event")
-		go c.updateEventMembers(e)
+		go c.removeEventMembers(e)
```

Comment:
It looks like `ReapInterval` is how often it tries to remove members who haven't been connected to since `ReconnectTimeout` which defaults to 24 hours.  I think we do want to consider members as failed if they leave or fail. This is consul's behavior: https://github.com/hashicorp/consul/blob/master/consul/serf.go#L44-L46